product-docs/installation/prometheus-metrics.md

```
@@ -0,0 +1,119 @@
+---
+description: >-
+  CISO Assistant exposes a /metrics endpoint in the Prometheus exposition format.
+  This page explains how to enable it and how to access it safely from Prometheus
+  without exposing it publicly.
+---
+
+# Prometheus metrics
+
+### Overview
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)